### PR TITLE
Add CKEditor on proposedSolution field on citizen project edit form

### PR DESCRIPTION
**Pull request by NyKo24**

### CODE REVIEW

File: src/Admin/CitizenProjectAdmin.php [hunk 1]
```diff
@@ -154,6 +154,8 @@ protected function configureFormFields(FormMapper $formMapper)
                 ])
                 ->add('proposedSolution', null, [
                     'label' => 'Solution du problème',
+                    'filter_emojis' => true,
+                    'attr' => ['class' => 'ck-editor'],
```
Comment (HeahDude) [inline src/Admin/CitizenProjectAdmin.php]: missing `'purify_html' => true`?

**Comment by B-Galati** [comment]
Rebase needed ?

File: src/Admin/CitizenProjectAdmin.php (hunk 1 above)
Comment (NyKo24) [reply src/Admin/CitizenProjectAdmin.php]: it's a WISIWIG not purify HTML it's needed no ?

Comment (HeahDude) [reply src/Admin/CitizenProjectAdmin.php]: Never trust the client.

**Pull request merged**